### DISCUSSION TITLE
now 150T， but Current difficulty 700+

### DISCUSSION THREAD

**Discussion by llin0903** [Chia-Network/chia-blockchain#6006] [Support]
I think this is abnormal. Can you tell me how to calculate the difficulty?

**Comment by qpirsel** [comment]
The difficulty is not your personal difficulty but the difficulty for everyone. Although I'm not aware of the formula I'm pretty sure that you can only increase everyone's (and so your) difficulty by each plot you make. What should decrease by each new of your plots is the estimated time to win.

**Comment by BrandtH22** [comment]
@llin0903 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
